textbook/12/2/normal.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>the distribution respectively -&gt; the distribution, respectively</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>